Add `osu! Malaysia Tournament X` wiki article (#12131)
ppy/osu-wiki: wiki/Tournaments/OMT/X/en.md

   - Eliminated players are free to enlist as replay providers for the later stages of the tournament in accordance to the [official tournament support guidelines](/wiki/Tournaments/Official_support#staff). 

### Round-specific rules

#### Qualifier rules

1. Each player will have to sign up to one of the Qualifier lobbies that have been scheduled and prepared by the tournament management in advance.
2. In the lobby, all players will have to consecutively play all of the eleven Qualifier beatmaps in the order of NM1 -> NM2 -> NM3 -> NM4 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2 -> DT3.
3. Players **are not allowed** to ban any beatmaps in the Qualifiers.
4. Players **are not allowed** to join (or register for) more than one Qualifier lobby.
5. Based on their performance in the Qualifier, players will be ranked based on their Z-score value across all eleven Qualifier beatmaps. A brief explanation on the concept of Z-score can be found [here](https://www.statisticshowto.com/probability-and-statistics/z-score/).
6. The 24 players with the **highest Z-score value** according to the above criteria will advance to the tournament proper as follows:

>    - Players in seeds #1 - #8 will be given an **automatic bye to the Round of 16**.
>    - Players in seeds #9 - #24 will **participate in the runoff brackets**.

Reviewer: ```suggestion
   - Players in seeds #1 – #8 will be given an **automatic bye to the Round of 16**.
   - Players in seeds #9 – #24 will **participate in the runoff brackets**.
```

7. Failure to attend in any of the predetermined Qualifier lobbies will result in an instant elimination from the tournament.

#### Runoff bracket rules

![](img/runoff-scheme.jpg "Runoff brackets diagram")

> 1. Based on their Qualifier seeding, players in seeds #9 - #24 will be placed in the runoff brackets according to the above scheme.

Reviewer: ```suggestion
1. Based on their Qualifier seeding, players in seeds #9 – #24 will be placed in the runoff brackets according to the above scheme.
```

   - Banned beatmaps will not be allowed to be picked by any players during the entire match.
   - Barring the tiebreaker, there are no restrictions as to which maps may or may not be protected or banned in a match.
3. Each player will be given a chance to pick **one warm-up beatmap** (up to 3:00 max) to be played in the lobby.
   - Playing a warm-up beatmap is not mandatory, and teams may elect to skip their warm-up pick should they wish to.
4. Players are expected to exercise common sense in pick time windows.
   - If a player is unable to come up with a pick within a 90-second time window of their picking turn, the pick will be given to the other player.
5. Unlike in other tournaments, OMT X will feature a dedicated pick for off-beat maps termed as the **EX/Special** pick. Maps in this pick will have a special playing condition attached to them (e.g. "The map has to be played with Easy") or be outright unconventional.
6. In a Free Mod pick, each player has to apply at least one mod to play the beatmap with. Allowed mods are Easy, Hard Rock, Hidden, or any possible combinations of the three mods. 

>    - Playing a FreeMod pick without any mods applied is not allowed.

Reviewer: ```suggestion
   - Playing a Free Mod pick without any mods applied is not allowed.
```
